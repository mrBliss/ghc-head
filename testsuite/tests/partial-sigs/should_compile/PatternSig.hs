{-# LANGUAGE PartialTypeSignatures #-}
module PatternSig where

bar :: Bool -> Bool
bar (x :: _) = True
