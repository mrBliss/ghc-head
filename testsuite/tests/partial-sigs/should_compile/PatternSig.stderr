TYPE SIGNATURES
  bar :: Bool -> Bool
TYPE CONSTRUCTORS
COERCION AXIOMS
Dependent modules: []
Dependent packages: [base, ghc-prim, integer-gmp]
